### ISSUE TITLE
could not read Password for 'https://***@github.com': No such device or address

### ISSUE THREAD

**Issue by Srikanth-Juluru** [platisd/bad-commit-message-blocker#14]
I am getting similar issue when I am using below public actions and its trying to clone the repository and its failing with same error. Could you please help me what is fix. 

internally - action  qcastel/github-actions-maven-release@master is trying to commit the code 

name: Release
uses: qcastel/github-actions-maven-release@master
env:
JAVA_HOME: /usr/lib/jvm/java-11-openjdk/
with:
git-release-bot-name: "tes"
git-release-bot-email: "[tes@cc.com](mailto:tes@cc.com)"
maven-args: "-Dmaven.javadoc.skip=true -DskipTests -DskipITs --settings settings.xml -Drepo.username=${{ secrets.ARTIFACTORY_CRED_USR }} -Drepo.password=${{ secrets.ARTIFACTORY_CRED_PAT }} "
access-token: ${{ secrets.GITHUB_TOKEN }}

Error : 

Error: 8,606 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-release-plugin:2.5.3:perform (default-cli) on project release_tag_poc: Unable to checkout from SCM
Error: 8,607 [ERROR] Provider message:
Error: 8,608 [ERROR] The git-clone command failed.
Error: 8,609 [ERROR] Command output:
Error: 8,609 [ERROR] Cloning into '/github/workspace/target/checkout'...
Error: 8,610 [ERROR] fatal: could not read Password for 'https://***@github.com': No such device or address
Error: 8,611 [ERROR] -> [Help 1]
Error: 8,611 [ERROR] 
Error: 8,612 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
Error: 8,612 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
Error: 8,612 [ERROR] 
Error: 8,612 [ERROR] For more information about the errors and possible solutions, please read the following articles:
Error: 8,612 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

**Comment by platisd** [comment]
Hi, please share your workflow file or a link to it.

**Comment by Srikanth-Juluru** [comment]
https://github.com/AAInternal/test-gitWorkflow/actions/runs/5323736049
above is URL for work flow. Also sharing the work flow file code.

```yaml
name: release-github-token

on:
  push:
    branches:
      - 828923-dotc-release-management

jobs:
  build:
    runs-on: ubuntu-latest
    if: "!contains(github.event.head_commit.message, '[ci skip]')"
    
   # permissions:
   #   contents: write
   #   pull-requests: write 
   #   issues: write
   #   packages: write

    steps:
      - uses: actions/checkout@v3     
            
      - name: Set up JDK 11
        uses: actions/setup-java@v3
        with:
            distribution: 'temurin'
            java-version: 11   

      - name: Release
        uses: qcastel/github-actions-maven-release@master
        env:
          JAVA_HOME: /usr/lib/jvm/java-11-openjdk/
        with:
          git-release-bot-name: "Srikanth-Juluru"
          git-release-bot-email: "Srikanth.Juluru@aa.com"
          maven-args: "-Dmaven.javadoc.skip=true -DskipTests -DskipITs --settings settings.xml -Drepo.username=${{ secrets.ARTIFACTORY_CRED_USR }} -Drepo.password=${{ secrets.ARTIFACTORY_CRED_PAT }}"
          access-token: ${{ secrets.GITHUB_TOKEN }}
          release-branch-name: 828923-dotc-release-management
```

**Comment by platisd** [comment]
I can't access that link and also I don't see how you are invoking `platisd/bad-commit-message-blocker` from your workflow file. Is something missing?

**Comment by platisd** [comment]
Or wait... Are you asking me how to debug a totally unrelated action?! :sweat_smile:

**Comment by Srikanth-Juluru** [comment]
Yes that correct, I saw similar issue reported one of your action so I am thinking you can help on different action. Really sorry if I confused you.
Is that possible to hemp me on this

**Comment by platisd** [comment]
Well, it is possible (almost everything is :shrug:) for me to help you on this, but I will not, since this will be extremely unproductive. After all, the issue you are facing is 99% unrelated to the one that was faced in this project and was resolved by 020647db1b6776da6e59d0f56d8565b6c. Instead, I will give you some free advice:

1. You should be asking your question in [qcastel/github-actions-maven-release](https://github.com/qcastel/github-actions-maven-release).
2. Debugging will be much easier if you **show your code** or a minimum example that reproduces the issue you have. Your code is currently in a private repository so no one else is able to see it. **You must help others help you.**
